docs/types.rst

```diff
@@ -355,6 +355,10 @@ Enums are one way to create a user-defined type in Solidity. They are explicitly
 to and from all integer types but implicit conversion is not allowed.  The explicit conversions
 check the value ranges at runtime and a failure causes an exception.  Enums needs at least one member.
 
+The data representation is the same as for enums in C: The options are represented by a sequence
```

Comment:
Strict sequence?

Reply:
What is a strict sequence? Perhaps "sequence of subsequent integers"? Any suggestions welcome...

Reply:
"Increasing sequence without gaps?"

Reply:
_a subsequent list of natural numbers_

Reply:
Thanks! I thought about 'subsequent sequence' first, but that didn't really cut it :)